mlflow/java/spark/pom.xml

```diff
@@ -0,0 +1,141 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>mlflow-spark</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
@smurching Should this read `1.4.1-SNAPSHOT`, or do we intend to start the new package at version 1.0?

Reply:
Oops yep it should be 1.4.1-SNAPSHOT, good catch!